install.sh

```diff
@@ -33,25 +33,62 @@ if [[ "${missing_npm}" -ne 0 ]]; then
 else
   echo -e "  NPM \xE2\x9C\x94"
 fi
+if [[ "${has_required_software}" == true ]]; then
+  echo -e "\033[0;32mSoftware requirements are met.\033[0m\n"
+else
+  echo -e "\033[0;31mSoftware requirements were not met. Please install the missing software above, then rerun this script.\033[0m" >&2
+  exit 1
+fi
 
-missing_mysql=$(mysql --version > /dev/null 2>&1; echo $?)
-if [[ "${missing_mysql}" -ne 0 ]]; then
-  echo -e "  MySQL \xE2\x9D\x8C" >&2
-  has_required_software=false
+# check if .env file exists and is correctly named
+# if a file is named env exists in the correct location, we rename it to .env
+has_required_env=true
+echo -e "\033[0;34mChecking if environment files exist\033[0m"
+
+# check for API .env
+file="./.env"
+badFile="./.env"
```

Comment:
```suggestion
file="./.env"
badFile="./env"
```

Reply:
Btw, would having a `env` file alongside a valid `.env` file affect the `dotenv` package? If it does, then your if-else structure is good. Otherwise, it'll be unnecessary to check for other files if we already have a valid `.env`.

Reply:
Nice catch. Thank you.